config.xml

```diff
@@ -59,15 +63,18 @@
         <package name="notepadpp.plugin.xmltools.vm"/>
         <package name="offvis.vm"/>
         <package name="onenoteanalyzer.vm"/>
+        <package name="pdbresym.vm"/>
         <package name="pdfstreamdumper.vm"/>
         <package name="pebear.vm"/>
         <package name="peid.vm"/>
         <package name="pesieve.vm"/>
         <package name="pestudio.vm"/>
         <package name="pkg-unpacker.vm"/>
         <package name="pma-labs.vm"/>
+        <package name="procdot.vm"/>
         <package name="processdump.vm"/>
         <package name="reg_export.vm"/>
+        <package name="regcool.vm"/>
```

Comment:
We should remove regcool as it is broken at the moment: https://github.com/mandiant/VM-Packages/issues/1004